Index: Test/test.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Test/test.py b/Test/test.py
--- a/Test/test.py	
+++ b/Test/test.py	(date 1681141968968)
@@ -1,8 +1,8 @@
 import pytest
 
-def setup_function(function):
+def setup_function():
     print(" setup ")
-def test_uno(function):
+def test_uno(pytestconfig):
     print("Test uno")
 
 def test_dos():
